api/src/main/java/org/pmiops/workbench/monitoring/attachments/MetricLabel.java

```diff
@@ -10,9 +10,12 @@
 public enum MetricLabel implements MetricLabelBase {
   BUFFER_ENTRY_STATUS("BufferEntryStatus", Enums.getValueStrings(BufferEntryStatus.class)),
   DATASET_INVALID("Invalid", Booleans.VALUE_STRINGS),
+  DATA_ACCESS_LEVEL("DataAccessLevel", Enums.getValueStrings(DataAccessLevel.class)),
+  GSUITE_DOMAIN(
+      "gsuite_domain"), // provide this (slightly redundant) in case we ever merge time series
```

Comment:
the formatter butchered this a bit, probably move the comment before the enum value instead